### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
- Create a app.rb file that will serve as your console app entry-point
         - Add method to add book
         - Add method to add person( teacher OR student )
         - Add method to add rental
         - Add method to list all books
         - Add method to list all persons
         - Add method  to list all rental to person
- Define the entry point in main.rb, this will be a method called main that is invoked at the end of your file

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,12 +1,125 @@
...
+        send(MENU_OPTIONS[option])
+      else
+        puts 'That is not a valid number'
+      end
+
+      break if option == '7' # Exit the loop if '7' is entered
+
+      puts ' '
+      puts 'Press any key to continue'
+      gets.chomp
```
Comment (danmainah) [inline main.rb]: - [ ] @Lucash2022 Great work on your UI however this function of requesting the user to press any key to continue is not part of the project, it adds an extra step to a beautiful and easy to use interface kindly consider removing it.

File: README.md [hunk 2]
```diff
@@ -88,6 +89,13 @@ Clone this repository to your desired folder:
 - Clone the project to your directory by running
```
Comment (danmainah) [inline README.md]: - [ ] @Lucash2022 Great work on your documentation, however, you have two sections describing how to clone the project kindly consider removing the first one just below the getting started title.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Lucash2022 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Linter setup correctly and all checks are passing

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 2 above)
Comment (Lucas-Erkana) [reply README.md]: @danmainah resolved this issue at this [commit](https://github.com/Lucash2022/School-Library/commit/74bc283b5ec8e9f25c0323b90595984aa7bf1339)

File: main.rb (hunk 1 above)
Comment (Lucas-Erkana) [reply main.rb]: @danmainah 
Thank you so much for pointing this out. I have resolved the issue at this [commit](https://github.com/Lucash2022/School-Library/pull/4/commits/d7053d1823710a8fcf2c35bb84f640ffbef81e03)

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Lucash2022,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**